jobserver/views/jobs.py

```diff
@@ -64,17 +65,25 @@ def get(self, request, *args, **kwargs):
         # have more than one backend-specific error code it makes sense to keep
         # things together.
         log_path = ""
+        log_path_url = ""
         if job.job_request.backend.slug == "tpp" and job.status_code == "nonzero_exit":
             log_path = os.path.join(
                 job.job_request.backend.parent_directory,
                 job.job_request.workspace.name,
                 "metadata",
                 f"{job.action}.log",
             )
+            log_path_url = (
+                furl(job.job_request.workspace.get_files_url())
+                / "metadata"
+                / f"{job.action}.log"
+            )
```

Comment:
Let's use make `log_path_url` be the result of calling `.url` on this structure.  I was surprised to see that property being called in the test since the variable ends in `_url`, since I had assumed it was the string, not a furl instance we were passing around.